### PR TITLE
Enhancement: Dynamic Filename + jsonl fix

### PR DESCRIPTION
**Pull request by SwiftPredator**
Dynamic filename + jsonl fix as proposed in #851. We also tried to download the file with the header extension which ended in a Network-Error. Since the code works and is a simple extension to the original code we think it's a good solution.

### CODE REVIEW

File: frontend/store/projects.js [hunk 1]
```diff
@@ -160,7 +160,7 @@ export const getters = {
       text: 'CSV'
     }
     const json = {
-      type: 'json',
+      type: 'json1',
```
Comment (fjen) [inline frontend/store/projects.js]: should be jsonl with an "L"

Comment (fjen) [reply frontend/store/projects.js]: (below too)

Comment (SwiftPredator) [reply frontend/store/projects.js]: Should I change it completely? Since in the codebase(see for example views.py line 377) its also defined as 'json1'.

Comment (fjen) [reply frontend/store/projects.js]: I think it's better to create a separate pull for this, since it would be out of scope here.

**Comment by fjen** [comment]
change message of last commit to be more precise. something like "replace json1 naming with jsonl"

**Comment by fjen** [comment]
first commit is a merge commit (2 parents). author information is wrong. please rebase.

**Comment by fjen** [comment]
works fine otherwise.

**Pull request closed without merging**